app/build.gradle

```diff
@@ -83,7 +83,16 @@ android {
             applicationIdSuffix ".debug"
             resValue "string", "provider_authority", "de.danoeh.antennapod.debug.provider"
             buildConfigField STRING, FLATTR_APP_KEY, mFlattrAppKey
-            buildConfigField STRING, FLATTR_APP_SECRET, mFlattrAppSecret
+            buildConfigField STRING, FLATTR_APP_SECRET, mFlattrAppSecret            
+            dexcount {
+                // Optionally, can disable Dexcount by setting the property
+                // in gradle.properties file or via -P command line argument.
```

Comment:
Now the comment does not really fit anymore. I think it is okay to remove it - the code itself is self-explanatory here